.circleci/README.md

```diff
@@ -0,0 +1,24 @@
+## CircleCI Project Maintenance Guideline
+
+### 1. Environment Variables
+
+Set environment variables on CircleCi by following [this guide](https://circleci.com/blog/new-on-circleci-import-project-environment-variables/).
+The CircleCi environment variables used in this project are:
```

Comment:
It would be nice if they are more specific.
Are you talking about environment variables in context and project ?
https://circleci.com/docs/2.0/env-vars/?gclid=Cj0KCQiA5dPuBRCrARIsAJL7oeg8WEHb3QGzxDNIPmdW95rAKRwhiOkXIKhqaba97o9QVCe-ubUMUnoaAlW0EALw_wcB#setting-an-environment-variable-in-a-context

If so, please make it something like this.
```
Please set environment variables in CircleCi as follows.
1. Set environment variables in context `scalar`
...

2. Set environment variables in the project
...
```